functions/src/backfillToTypesenseFromFirestore.js

```diff
@@ -40,8 +40,16 @@ module.exports = functions.handler.firestore.document
         return false;
       }
 
-      const querySnapshot =
-        await admin.firestore().collection(config.firestoreCollectionPath).get();
+      const isSubcollection = config.firestoreCollectionPath.includes("/");
```

Comment:
Is there a case where a collection path can include `/` and it's not a sub-collection? 

For eg, let's say a set of admin users are nested under `users/admin`, and regular users are nested under `users/regular`?

Reply:
This cannot be the case. users/admin and users/regular are not valid subcollections. In that example there would be a document with the id admin under the users collection and a document with the id regular under the users collection. The correct syntax for subcollections would be users/{userId}/admin and users/{userId}/regular